airflow/models/taskinstance.py

```diff
@@ -264,13 +263,13 @@ def clear_task_instances(
 
         drs = (
             session.query(DagRun)
-            .filter(
+                .filter(
```

Comment:
It looks like the static checks weren't happy with these indentations?

Reply:
Unintentional change - reverted